include/zephyr/bluetooth/audio/audio.h

```diff
@@ -2020,7 +2021,7 @@ int bt_audio_unicast_group_create(struct bt_audio_unicast_group_param *param,
  *  @return 0 in case of success or negative value in case of error.
  */
 int bt_audio_unicast_group_add_streams(struct bt_audio_unicast_group *unicast_group,
-				       struct bt_audio_unicast_group_stream_param params[],
+				       struct bt_audio_unicast_group_stream_pair_param params[],
 				       size_t num_param);
```

Comment:
Should this be renamed to `add_stream_pairs`?

Reply:
Not necessarily, as user might not pair the streams.